packages/plugin-core/src/commands/ShowLegacyPreview.ts

```diff
@@ -0,0 +1,26 @@
+import _ from "lodash";
+import { DENDRON_COMMANDS } from "../constants";
+import { VSCodeUtils } from "../utils";
+import { MarkdownUtils } from "../utils/md";
+import { BasicCommand } from "./base";
+
+type CommandOpts = {};
+type CommandOutput = any;
+
+export class ShowLegacyPreviewCommand extends BasicCommand<
+  CommandOpts,
+  CommandOutput
+> {
+  key = DENDRON_COMMANDS.SHOW_LEGACY_PREVIEW.key;
+  async sanityCheck() {
+    if (_.isUndefined(VSCodeUtils.getActiveTextEditor())) {
+      return "No document open";
+    }
+    return;
+  }
+
+  async execute(_opts?: CommandOpts) {
+    // eslint-disable-next-line  no-return-await
+    return await MarkdownUtils.openPreview();
```

Comment:
we should be able to change this - show legacy preview should just open markdown-preview-enhanced at this point